VERSION

```diff
@@ -1 +1 @@
-5.23.2
+5.22.3
```

Comment:
I think the two `Podfile.lock`s for the examples should also be updated in line with this.